src/home/homed-home.c

```diff
@@ -211,6 +211,7 @@ Home *home_free(Home *h) {
 
         h->ref_event_source_please_suspend = sd_event_source_disable_unref(h->ref_event_source_please_suspend);
         h->ref_event_source_dont_suspend = sd_event_source_disable_unref(h->ref_event_source_dont_suspend);
+        h->inhibit_suspend_event_source = sd_event_source_disable_unref(h->inhibit_suspend_event_source);
```

Comment:
so the `ref` prefix was supposed to indicate that this is backed by a "reference" pipe, i.e. a pipe we use as reference counter.

i think we should stick to the `ref_` prefix for the suspend inhibitor too, since it uses the same mechanism

Reply:
hence maybe name:

```ref_event_source_inhibit_suspend`

Reply:
The inhibitor doesn't actually work as a reference counter on the home area, thus I didn't prefix it with `ref_`

Reply:
what do you mean by that? the pipe that this inhibitor is encapsulated by is a ref counting concept: we want that we can hand out the fd many times, and once the last peer closes its fd we see the EOF and act on it.

Reply:
Ah sorry I might have misunderstood what you mean.

So the `ref_` _isn't_ referring to the fact that this reference counting pipe is keeping the home area activated, and is instead just referring to the fact that this event source is for a pipe that's being used to reference count something? This isn't really clear from the code. I thought the `ref_` prefix meant that this event source, if not null, keeps reference count for the whole home area. In other words, it's the event source for the FDs returned by `Ref()` and/or `Acquire()`, So, since the inhibitor FD doesn't actually reference the home area, I didn't add the `ref_` prefix

Reply:
well, it meant *both* ;-)

but i guess this is bikeshedding, let's just merge this